code/modules/projectiles/projectile/magic.dm

```diff
@@ -262,7 +262,7 @@ proc/wabbajack(mob/living/M)
 				S.icon = change.icon
 				if(H.mind)
 					H.mind.transfer_to(S)
-					S << "You are an animated statue. You cannot move when monitored, but are nearly invincible and deadly when unobserved! Do not harm [firer.name], your creator."
+					S << "<span class = 'userdanger'>You are an animated statue. You cannot move when monitored, but are nearly invincible and deadly when unobserved! Do not harm [firer.name], your creator.</span>"
```

Comment:
Dunno if byond supports `<u>` but if it does, might be a good idea to underline `do not harm [firer.name]`